test/deploy-test.ts

```diff
@@ -103,21 +108,30 @@ class MockDeployEffects extends MockAuthEffects implements DeployEffects {
     this.deployConfig = config;
   }
 
-  *visitFiles(path: string) {
-    yield* visitFiles(path);
+  *visitFiles(path: string, {ignoreObservable = true} = {}) {
+    yield* visitFiles(path, {ignoreObservable});
```

Comment:
We probably shouldn’t repeat the default here (in case it changes). Would this work instead?

```suggestion
  *visitFiles(path: string, options?: {ignoreObservable?: boolean}) {
    yield* visitFiles(path, options);
```